### ISSUE TITLE
Put types into a namespace instead of module

### ISSUE THREAD

**Issue by xperiandri** [Zaid-Ajaj/Snowflaqe#33]
Putting types into a module is not a good idea. As it causes some issues with their consumption from WPF/UWP/Uno

**Comment by Zaid-Ajaj** [comment]
Hi @xperiandri, can you tell me more about this issue? The global types are included as part of a namespace and the query-specific types are in a module _inside_ that same namespace

**Comment by xperiandri** [comment]
You cannot instantiate nested types in XAML. For example to use as sample data to display on designer

**Comment by Zaid-Ajaj** [comment]
I see, that makes sense. Let me try to see what I can do

**Comment by xperiandri** [comment]
Is it fixed?

**Comment by Zaid-Ajaj** [comment]
Nope, didn't have time for it. Feel free to submit a PR for it 😄 it should be another boolean option. Maybe "nestedModuleTypes"? 🤔 where true is default, false -> generates namespaces instead

**Comment by xperiandri** [comment]
Could you point to the line where it happens?

**Comment by Zaid-Ajaj** [comment]
It happens here in the function `CodeGen.generateQualifiedModule`:
https://github.com/Zaid-Ajaj/Snowflaqe/blob/master/src/Program.fs#L421
